website/api-gateway/src/json-rpc/json-rpc-method.handler.ts

```diff
@@ -23,28 +12,22 @@ import {
   testBalance,
 } from '../kafka/kafka-events';
 import { KafkaParams } from '../kafka/types';
+import { RpcMapping } from './types';
+
+export function jsonRpcMethodHandler(method: KAFKA_TOPICS | string, params: KafkaParams): Promise<any> {
+  const getRpcMethod: RpcMapping = {
+    [KAFKA_TOPICS.PROGRAM_DATA]: programData,
+    [KAFKA_TOPICS.PROGRAM_ALL]: programAll,
+    [KAFKA_TOPICS.PROGRAM_META_ADD]: programMetaAdd,
+    [KAFKA_TOPICS.PROGRAM_META_GET]: programMetaGet,
+    [KAFKA_TOPICS.PROGRAM_ALL_USER]: programAllUsers,
+    [KAFKA_TOPICS.MESSAGE_ALL]: messageAll,
+    [KAFKA_TOPICS.MESSAGE_DATA]: messageData,
+    [KAFKA_TOPICS.MESSAGE_ADD_PAYLOAD]: messageAddPayload,
+    [KAFKA_TOPICS.TEST_BALANCE_GET]: testBalance,
+  };
```

Comment:
Don't you think that it's better to make this variable outside of the function? And maybe without `get` in naming